lib/legato/management/account_summaries.rb

```diff
@@ -0,0 +1,42 @@
+module Legato
+  module Management
+    class AccountSummaries
+
+      extend Finder
+
+      def self.default_path
+        "/accountSummaries"
+      end
+
+      def path
+        "/accountSummaries"
+      end
+
+      attr_accessor :user, :account, :summary_web_properties, :summary_profiles
+
+      def initialize(attributes, user)
```

Comment:
Overall, I think too much work is being done in this initialize method. I think it could be broken out into each subsequent class in the tree, and then used to build the tree structure.

I would suggest something like this:

```ruby
def account
  @account ||= Account.build_from_summary(attributes, user)
end
```

Then the `Account.build_from_summary` will create its set of web_properties by mapping over attributes and calling `WebProperty.build_from_summary`.

Then the `WebProperty` does the same pattern with the profile attributes and `Profile`.